lib/Gocdb_Services/NotificationService.php

```diff
@@ -137,4 +139,35 @@ private function sendEmail($roleRequested, $requestingUser, $entityName, $approv
 
         \Factory::getEmailService()->send($emailAddress, $subject, $body, $headers);
     }
+
+    private function sendGocdbAdminsEmail($roleRequested, $requestingUser) {
+        $subject = sprintf(
+            'GOCDB: A Role request from %1$s %2$s over %3$s has no approving users',
+            $requestingUser->getForename(),
+            $requestingUser->getSurname(),
+            $roleRequested->getOwnedEntity()->getName()
+        );
+
+        $body = sprintf(
+            implode("\n", array(
+                'Dear GOCDB Admins,',
+                '',
+                '%1$s %2$s requested the "%3$s" role over %4$s, and there are no '.
+                'approving users available for the request.',
+                '',
+                'You can approve or deny the request here:',
+                '    %5$s/index.php?Page_Type=Role_Requests',
+            )),
+            $requestingUser->getForename(),
+            $requestingUser->getSurname(),
+            $roleRequested->getRoleType()->getName(),
+            $roleRequested->getOwnedEntity()->getName(),
+            $this->getWebPortalURL()
+        );
+
+        $emailAddress = "gocdb-admins@mailman.egi.eu";
```

Comment:
Could this email address be made configurable? similar to this change: https://github.com/GOCDB/gocdb/commit/49b5db6054eedc59fcdf9a1e7497674b59ae551c

Reply:
Thanks, changes made